src/Extensions/MemberExtension.php

```diff
@@ -41,6 +54,28 @@ public function updateCMSFields(FieldList $fields)
                     ->addComponent(GridFieldRevokeLoginSessionAction::create())
             )
         );
+        # todo: remove end //
+
+        $fields->removeByName(['SessionManagerField']);
+
+        if (!$this->owner->exists() || !$this->currentUserCanViewSessionManagerConfig()) {
+            return $fields;
+        }
+
+        $fields->addFieldToTab(
+            'Root.Main',
+            $methodListField = SessionManagerField::create(
```

Comment:
I think this is leftover naming from MFA?

```suggestion
            $sessionManagerField = SessionManagerField::create(
```